src/utils/EnvVars.ts

```diff
@@ -14,8 +46,28 @@ export default {
 
     CAPTAIN_IS_DEBUG: !!process.env.CAPTAIN_IS_DEBUG,
 
+    DEBUG_SOURCE_DIRECTORY: process.env.DEBUG_SOURCE_DIRECTORY,
+
+    CAPROVER_IMAGE: process.env.CAPROVER_IMAGE,
+
+    SHOW_DOCKER_COMMANDS: !!process.env.SHOW_DOCKER_COMMANDS,
+
+    // Internal container ports, these does not need to change.
```

Comment:
These 3, as you mentioned in the comment, do not need to be modified. So let's remove them for the sake of simplicity.

Reply:
Ok, I will remove them. The ideea was to have a constant for everywhere 80,443 and 3000 show up as magical values not to confuse them with the external ones that can be changed. But agree it's fine to be conservative.